libs/ledger-live-common/src/families/elrond/js-getTransactionStatus.ts

```diff
@@ -134,9 +134,10 @@ const getTransactionStatus = async (
   }
 
   if (!errors.amount && totalSpentEgld.gt(a.spendableBalance)) {
-    errors.amount = tokenAccount
-      ? new NotEnoughBalanceInParentAccount()
-      : new NotEnoughBalance();
+    errors.amount =
+      tokenAccount || !["delegate", "send"].includes(t.mode)
```

Comment:
Are you sure about this? Means "_if the tx is neither a delegate or a send, use the `NotEnoughBalanceInParentAccount` error_", but it doesn't seem related at all?

Reply:
I think we need another error here, something like "Not enough EGLD to pay the transaction fees".

Reply:
Done